tests/puppeteer/pages/FO/home.js

```diff
@@ -19,11 +22,39 @@ module.exports = class Home extends CommonPage {
   }
 
   /**
-   * Go the product page
+   * Go to the product page
    * @param id, product id
    */
   async goToProductPage(id) {
     await this.page.waitForSelector(this.logoHomePage, {visible: true});
     await this.waitForSelectorAndClick(this.productImg.replace('%NUMBER', id), 5000);
   }
+
+  /**
+   * Get number of products displayed in all products page
+   * @return integer
+   */
+  async getNumberOfProducts() {
+    const productNumber = await this.getTextContent(this.totalProducts);
+    const numberOfProduct = /\d+/g.exec(productNumber.match(/Il y a ([0-9]+)/)).toString();
+    return parseFloat(numberOfProduct);
```

Comment:
```suggestion
    return parseInt(numberOfProduct);
```
I think we'll never have a float value for a number of product :-P

Reply:
Yess yes sorry :D 
@boubkerbribri please fix your function also

Reply:
The difference between parseInt and parseFloat
parseInt can interpret text as hexadecimal
parseFloat will nly parse to base-10